src/puppy/platforms/win32/windefs.nim

```diff
@@ -79,22 +86,22 @@ proc WinHttpConnect*(
 ): HINTERNET {.dynlib: "winhttp".}
 
 proc WinHttpOpenRequest*(
```

Comment:
Things look close to passing tests, close to done here which is gerat. I think the remaining issue is that WinHttpOpenRequest needs the full correct function parameters back (they're deleted currently in the PR).

See https://github.com/treeform/puppy/runs/7845915480?check_suite_focus=true